components/logo/README.md

```diff
@@ -4,18 +4,67 @@
 
 ## Toegankelijkheid
 
-Soms is het logo de belangrijkste manier om te zien op welke website je bent, en wat de naam van de organisatie is. Zorg dan dat de tekst in het logo groot genoeg is en het contrast voldoende is, anders is de afzender niet duidelijk. WCAG maakt een uitzondering voor logo's, maar dat is geen goede reden om een onduidelijke site te hebben.
+Test je logo ook in _dark mode_ en in in _forced colors_ mode.
 
 ## Bestandsformaat
 
 Gebruik voor het logo bij voorkeur een een vectorafbeelding zoals SVG, zodat het logo scherp is op verschillende soorten schermen. Een pixel-logo (zoals PNG) is vaak onscherp op moderne schermen die een hoge pixeldichtheid hebben.
 
-Het is mogelijk een scherpe versie van een pixelafbeelding te hebben voor schermen met een hoge pixeldichtheid. Gebruik daarvoor [`srcset` met een onder andere een `2x` variant](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset). SVG gebruiken is meestal toch handiger, omdat je maar één bestand nodig hebt voor alle soorten pixeldichteid.
+Het is mogelijk een scherpe versie van een pixelafbeelding te hebben voor schermen met een hoge pixeldichtheid. Gebruik daarvoor [`srcset` met een onder andere een `2x` variant](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset). SVG gebruiken is meestal toch handiger, omdat je maar één bestand nodig hebt voor alle soorten pixeldichtheid.
```

Comment:
Misschien kunnen we dit toevoegen:
- Opmerking: Als je SVG’s als afbeeldingsformaat gebruikt, test je werk dan in IE11 als die browser nog ondersteund wordt door je organisatie. (Zie CanIUse)
- Vergeet niet om het logo en de tekst te controleren door in te zoomen tot 200% en tot 400% terwijl de viewportbreedte 1280px is voor leesbaarheid en overlappingsproblemen